bitergia.tf

```diff
@@ -0,0 +1,186 @@
+variable "bitergia-db_password" {}
```

Comment:
I do this to keep it consistent with the AWS name ("bitergia-db"). It's also done for the other DB.